clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -164,9 +170,10 @@
     private final Thread thread;
 
     /**
-     * True if this client is closed.
+     * During a close operation, this is the time at which we will time out all pending operations
+     * and force the RPC thread to exit.  If the admin client is not closing, this will be 0.
```

Comment:
"will be 0" should probably be "should be INVALID_SHUTDOWN_TIME". I can fix this when merging though.